### DISCUSSION TITLE
I Created A Documentation

### DISCUSSION THREAD

**Discussion by JenuelDev** [jenssegers/laravel-mongodb#2161] [Show and tell]
Does anyone want to contribute to the documentation of laravel MongoDB? check it here: https://github.com/BroJenuel/laravel-lumen-mongodb-docs

check live: https://laravel-mongodb.netlify.app/

I used nuxt content so it will be easy.

For the initial, I copied the data in the readme.

**Comment by divine** [comment]
Looks great!
